### DISCUSSION TITLE
how to insert a photo into github-pages!?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#52434] [Other Feature Feedback, Questions, & Ideas] [answered]
how to insert a photo into github-pages!?

how to add a photo to the github - page. i guess that i have to load it up to the github repository

what is known here: heard that we can ad a image while adding a  ! and the alternative text with [ ] all is enclosed with ().

![Screenshot of a comment on a GitHub issue showing an image, added in the Markdown, of an Octocat smiling and raising a tentacle.](https://myoctocat.com/assets/images/base-octocat.svg)

https://docs.github.com/de/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images

example:  lets say we want to add this photo to the github-page
https://imgur.com/a/mpkZiB1

how to proceed

**Comment by p0lygun** [comment]
if you have pages set up why not put the image in repo and use the direct URL to it ?

**Reply by fsj-digital** [reply]
hello dear polyqun many thanks for that. awesome - well this is a great idea. 
how would this could go - i add the image to the reoo and then i link to the repo 

btw- what about the quick and dirty example if i use this path #

: lets say we want to add this photo to the github-page
https://imgur.com/a/mpkZiB1

does this work too!?


look forward to hear from you dear pOlygun

**Comment by fsj-digital** [comment]
inserted a image 

can see it here: https://github.com/fsj-digital/DigitalHub/blob/main/README.md 

but not here:  https://fsj-digital.github.io/DigitalHub/

btw; this is the insert-markdown: 

![de:hub.de-Projekt](https://github.com/fsj-digital/DigitalHub/blob/main/de_hub.jpg)

question: how to size it down a bit - in other words how to make it a bit smaller!?

any and all ideas and help gladly appreciated...

**Comment by airtower-luna** [comment, marked as answer]
With the image in your repository a relative URL should work, with the example in your README this should work (because both files are in the same directory):

```
![de:hub.de-Projekt](de_hub.jpg)
```

You should work on the alt text (the part in the square brackets) though. "de:hub.de-Projekt" won't tell someone who can't see the picture (e.g. because they're blind and using a screen reader) what it's about.

**Comment by fsj-digital** [comment]
hello and good day  

many many thanks  this works . Awesome. 
- one question - how to rework the size - can i resize the size of the image a bit!?
love to hear  from you 

regards

**Comment by airtower-luna** [comment]
There's nothing in the Markdown syntax to set image size.

You can try to use an HTML `<img>` tag instead. GitHub-flavored Markdown allows most HTML tags, but I'm not sure if they'd filter the `width` or `height` attributes.
